web-backend/src/main/resources/application.yml

```diff
@@ -41,4 +41,7 @@ gg:
 # Application specific properties
 vms:
   system-email: test@email.com
-  admin-email: test@email.com
\ No newline at end of file
+  admin-email: test@email.com
+
+mail:
+  url: http://localhost:${wiremock.server.port}/v1.0/users/%s/sendMail
```

Comment:
This signals a problem with our Wiremock setup, basically its not properly reusable. Wiremock supports the concept of defining stubs in resources so they can be reused across the application. This URL is the same as the MS Graph URL, there has to be a cleaner way of doing this.

Reply:
For now let's go ahead with this.